packages/blade/src/components/Amount/__tests__/Amount.web.test.tsx

```diff
@@ -0,0 +1,146 @@
+import { Amount } from '../Amount';
```

Comment:
Same issues as the ones highlighted in the native test. Description errors, etc.